packages/apps/halo-app/src/components/AlertDialog/AlertDialog.tsx

```diff
@@ -6,9 +6,7 @@ import { Transition } from '@headlessui/react';
 import * as AlertDialogPrimitive from '@radix-ui/react-alert-dialog';
 import React, { cloneElement, ComponentProps, Fragment, ReactHTMLElement, ReactNode, useEffect, useState } from 'react';
 
-import { defaultDescription, defaultFocus } from '../../styles';
-import { mx } from '../../util';
-import { Input, InputProps } from '../Input';
+import { Input, InputProps, defaultDescription, defaultFocus, mx } from '@dxos/react-ui';
```

Comment:
Alert dialog can't live in `@dxos/react-components`?

Reply:
I have a very complicated answer for this simple question 😅 — I want to redevelop this to be less specific to HALO’s use-case, so for now I gave the component to HALO.